webClient/src/app/shared/editor-control/editor-control.service.ts

```diff
@@ -53,7 +53,9 @@ export class EditorControlService implements ZLUX.IEditor, ZLUX.IEditorMultiBuff
   public closeAllFiles: EventEmitter<string> = new EventEmitter();
   public undoCloseAllFiles: EventEmitter<string> = new EventEmitter();
   public activeDirectory = '';
+  public refreshFileMetadatdaByPath: EventEmitter<string> = new EventEmitter();
   public deleteFile: EventEmitter<string> = new EventEmitter();
+  public refreshFileMetadatdaByPath: EventEmitter<string> = new EventEmitter();
```

Comment:
You added a duplicate here by accident (maybe from Git cleanup things)

Reply:
Fixed